src/test/java/com/microsoft/azure/relay/SendReceiveTest.java

```diff
@@ -204,6 +204,24 @@ public void httpLargePOSTAndLargeResponseTest() throws IOException {
         httpRequestSender("POST", largeStr, largeStr);
 	}
 	
+	@Test
+	public void httpWriteSmallThenSmallResponseTest() throws IOException {
```

Comment:
>httpWriteSmallThenSmallResponseTest [](start = 13, length = 35)

Are there any tests which write some bytes to the response, pausing allowing the flush timer to send those bytes, then send some more bytes in the response body and validate the exact number of bytes and values are transmitted to the client?